src/components/app/ProjectPreviewSet.svelte

```diff
@@ -43,12 +45,13 @@
     }
 
     let listed = $derived(sortProjects(set).filter((p) => p.isListed()));
+    const user = getUser();//Amy
 </script>
 
 <div class="projects">
     {#each listed as project (project.getID())}
         {@const removeMeta = remove(project)}
-        <ProjectPreview {project} link={project.getLink(true)}
+        <!--amy--><ProjectPreview {project} link={project.getLink(true)} anonymize={isAudience($user, project)} 
```

Comment:
It's closer, but this line is wrong:

* If a `ProjectPreviewSet` is rendered on a gallery page, it should be anonymized only when the user is not a curator or creator on the gallery (because those in a gallery should be able to see others in the gallery). If they aren't, then it should be anonymous.
* If a `ProjectPreviewSet` is on the `/projects` page, it should never be anonymous, because that page shows all of the logged in person's projects. One could argue that they shouldn't show the person logged in though, because all of them would show that.

Because the `ProjectPreviewSet` needs to behave differently depend on where it is shown, it needs an `anonymized` prop as well, and all places they are created should meet the requirements above.